CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [0.5.0] (2024-05-15)
```

Comment:
The link definition for this headline is missing.

I'm in favor of not linking headlines anymore. What do you think?

Reply:
I think we should adopt that if we agree on next release with your changelog automation in place. Fixing this now.